### PR TITLE
feat: Migrate to `TrieAccount` of alloy-trie

### PR DESCRIPTION
**Pull request by moricho**
## Motivation
`Account` has been migrated to `TrieAccount` of  alloy-trie in https://github.com/alloy-rs/trie/pull/65

## PR Checklist

- [ ] Added Tests
- [ ] Added Documentation
- [ ] Breaking changes

### CODE REVIEW

File: crates/consensus/src/lib.rs [hunk 1]
```diff
@@ -9,8 +9,7 @@
 
 extern crate alloc;
 
-mod account;
-pub use account::Account;
+pub use alloy_trie::TrieAccount as Account;
```
[the remark this answers is not in the document]
Comment (mattsse) [reply crates/consensus/src/lib.rs:12]: can we also re-export TrieAccount and mark this rename as deprecated?

**Review by mattsse — CHANGES_REQUESTED** [review]
awesome, one nit

then we also want the conversion for genesis account seperately

**Review by mattsse — APPROVED** [review]

**Comment by moricho** [comment]
@mattsse I just saw your comment. Thank you for handling the genesis account conversion!

**Comment by mattsse** [comment]
thank you!

I quickly pushed this over the line because we needed a new release on reth

**Pull request merged**